src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/mainscreen/viewmodel/MainScreenViewModel.kt

```diff
@@ -54,7 +54,7 @@ class MainScreenViewModel : ViewModel() {
     }
 
     private fun setActiveChunkText(chunk: Chunk) {
-        selectedChunkTitle.set(ContentLabel.of(chunk.contentType).value.toUpperCase())
+        selectedChunkTitle.set(messages["verse"].toUpperCase())
```

Comment:
you couldn't do:
```selectedChunkTitle.set(messages[ContentLabel.of(chunk.contentType).value.toUpperCase()])```
?

Reply:
Probably. Something makes me uncomfortable about the underlying assumptions about ContentType and ContentLabel that we know should result in `messages["verse"]` in this case. But we did just do a refactor on these classes, and as long as they are not changing it's probably fine. Sorry if I'm making this a bigger deal than it is!